src/common/webrtc/io.rs

```diff
@@ -251,6 +251,16 @@ pub struct WebRtcTransport {
     rx: smol::channel::Receiver<IoPkt>,
 }
 
+impl Drop for WebRtcTransport {
+    fn drop(&mut self) {
+        let _ = self.stun.tx.close();
```

Comment:
would it be cleaner / possible to implement `close` for `IoPktChannel`?

Reply:
IoPktChannel are cloned and dropped during the lifetime of the WebRTC connection so there is s a risk to close channel early. Doing it with WebRtcTransport is cleaner since it goes out of scope at the same time at WebRtcApi thus helps us closes the spawned tasks properly